benchmarks/request_trace/convert_to_perfetto.py

```
@@ -435,6 +429,91 @@ def _prepare_tool_items(tool_records: list[dict[str, Any]]) -> list[dict[str, An
+        and record["tool"].get("tool_call_id")
+    }
+
+    by_trajectory: dict[tuple[str, str], list[dict[str, Any]]] = {}
+    for item in request_items:
+        by_trajectory.setdefault((item["session_id"], item["trajectory_id"]), []).append(
+            item
+        )
+
+    for requests in by_trajectory.values():
+        requests.sort(key=lambda item: item["ts_us"])
+        for request_item, next_request_item in zip(requests, requests[1:]):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)